src/plugins/collapsible-alerts/collapsible-alerts.js

```diff
@@ -27,9 +27,11 @@ var componentName = "wb-collapsible",
 		// Start initialization
 		// returns DOM object = proceed with init
 		// returns undefined = do not proceed with init (e.g., already initialized)
-		details = wb.init( event, componentName, selector );
+		var details = wb.init( event, componentName, selector ),
```

Comment:
```details``` variable is already declared at line 19. 

You might want to remove the declaration at line 19 and declare the variable again in the declaration block on line 75